src/main/java/wolox/training/models/Book.java

```diff
@@ -78,6 +81,7 @@ public String getYear() {
     }
 
     public void setYear(String year) {
+        Preconditions.checkNotNull(year, "year must be not null");
```

Comment:
Add the following validations:

- That the field is not empty

- That the field only contains numerical values